### ISSUE TITLE
Unvalidate/Disallow Strings with FormValidation.io

### ISSUE THREAD

**Issue by pienisieni** [formvalidation/support#727]
Am I missing something or why FormValidation.io can't be used to disallow/unvalidate strings with regex? For example &mdash; I would like to prevent users from filling in URLs to the textarea field. This is a very efficient way to prevent spam.

So the opposite to usual behavior, &mdash; if formvalidation finds a string inside a field, it would trigger the invalid error. But a field not containing the string would validate.

Is this not possible?

**Comment by nghuuphuoc** [comment]
Have you take a look at the [regexp](http://formvalidation.io/validators/regexp/) validator?

**Comment by pienisieni** [comment]
yes -- that's the first thing i looked at. and afaik it's only used to require fields. i wish to do the opposite. allow everything except regex defined strings.
